draft-bbs-signatures.md

```diff
@@ -298,20 +298,20 @@ Parameters:
 
 Definitions:
 
-- HKDF-Extract is as defined in [@!RFC5869], instantiated with hash H.
-- HKDF-Expand is as defined in [@!RFC5869], instantiated with hash H.
+- HKDF-Extract is as defined in [@!RFC5869], instantiated with hash function HASH.
+- HKDF-Expand is as defined in [@!RFC5869], instantiated with hash function HASH.
 - I2OSP and OS2IP are as defined in [@!RFC8017], Section 4.
 - L is the integer given by ceil((3 * ceil(log2(q))) / 16).
-- "BBS-SIG-KEYGEN-SALT-" is an ASCII string comprising 20 octets.
+- INITSALT is the octet string "0x4242532d5349472d4b455947454e2d53414c542d", obtained by taking the UTF-8 representation of the ASCII string "BBS-SIG-KEYGEN-SALT-".
```

Comment:
I think its great to have to octet string here explicitly however I was also thinking that we could update the encoding of elements to be hashed section to include more generally how string based values should be encoded prior to hashing?

Reply:
Do you think you could address this in this PR?

Reply:
If so I would update this section as follows.

```suggestion
- INITSALT the string value of "BBS-SIG-KEYGEN-SALT-", for the avoidance of doubt as per the rules outlined in (#encoding-of-elements-to-be-hashed) the octet string based representation past to the `HASH` function is "0x4242532d5349472d4b455947454e2d53414c542d".
```

Reply:
I didn't see other usage of ASCII strings in the text; all other references (unless I missed something?) take octet string as input (aka byte arrays, right?), which made me think that encoding for these were application-specific.

Reply:
Right, good catch, I have noted below I think the CipherSuite_ID input value type should be updated to be of type string and therefore would use this rule

Reply:
> If so I would update this section as follows.

If we had the ASCII encoding rules to the hash section (which I'll do), I don't think having the explicit value is necessary in that section (feels repetitive? and a precedent to doubly-explain hash procedure everywhere else..)